Commands/InteractionServiceHandler.cs

```diff
@@ -38,7 +39,7 @@ public static async Task RegisterModulesAsync()
 		// The service might not have been initialized yet
 		if (_interactionService == null)
 		{
-			await Program.LogAsync("InteractionServiceManager.RegisterModulesAsync()", "InteractionService not initialized yet", LogSeverity.Error);
+			Log.Error("InteractionService not initialized yet, InteractionServiceManager.RegisterModulesAsync()");
```

Comment:
I think providing details about where the error comes from is a bit unnecessary, I'm not sure why I did that really lol
What do you think?

```suggestion
			Log.Error("Failed to register modules: InteractionService not initialized.)");
```

Reply:
Yes, I also think it is unnecessary.